### DISCUSSION TITLE
iOS 17  PWA possible ?

### DISCUSSION THREAD

**Discussion by user858753257** [schlagmichdoch/PairDrop#184] [Q&A]
Is there a way for that ? 

When I share it from Safari to Homescreen safari opens a new tab

**Comment by schlagmichdoch** [comment]
What’s new for PWAs in iOS17? Do you have a link where I can find out more?

**Reply by user858753257** [reply]
Sorry I meant Safari 17 with web push .
https://dev.to/this-is-learning/safari-17-beta-a-game-changer-for-pwas-on-macos-ddp

BTW:
How can I add the PWA on iPhone ?

**Reply by schlagmichdoch** [reply]
Sorry, I have somehow missed your answer!

I will implement standalone display mode in the next minor version `v1.11.0` next week. Not sure yet how to implement web push and whether its useful if we work on a native app. Feel free to create an enhancement issue for it.

About your other question: Just visit https://pairdrop.net/ with your iPhone, tap the share icon in Safari and select "Add to Homescreen": https://www.cdc.gov/niosh/mining/content/hearingloss/installPWA.html
